functions.js

```diff
@@ -18,7 +18,11 @@ OUTPUT:
 ]*/
 
 export function getDogs(arr) {
-    return [];
+    const getDogsArr = arr.filter((pet) => {
+        if(pet.type === 'dog')
+            return true;
+    });
+    return getDogsArr;
 }
 
```

Comment:
Great solution!  Think about how you could do this with fewer lines of code.  (implicit returns, etc.).

Reply:
something like this?!?
       if(pet.type === 'dog' => true);
return getDogsArr;